lib/config/config-file.js

```diff
@@ -472,13 +498,14 @@ function resolve(filePath, relativeTo) {
     let normalizedPackageName;
 
     if (filePath.indexOf("plugin:") === 0) {
+        const pluginPath = filePath;
         const packagePath = filePath.substr(7, filePath.lastIndexOf("/") - 7);
         const configName = filePath.substr(filePath.lastIndexOf("/") + 1, filePath.length - filePath.lastIndexOf("/") - 1);
 
         normalizedPackageName = normalizePackageName(packagePath, "eslint-plugin");
         debug(`Attempting to resolve ${normalizedPackageName}`);
         filePath = resolver.resolve(normalizedPackageName, getLookupPath(relativeTo));
-        return { filePath, configName };
+        return { filePath, configName, pluginPath };
```

Comment:
Is there a reason to include `pluginPath` in the return value when it's always the same as `filePath`?

According to the JSDoc comment, the return value of this function is "A path that can be used directly to load the configuration", but it would be useful if it described the properties of the return value in more detail.